mono/metadata/tabledefs.h

```diff
@@ -164,6 +164,7 @@ enum {
 #define METHOD_IMPL_ATTRIBUTE_NOOPTIMIZATION       0x0040
 #define METHOD_IMPL_ATTRIBUTE_MAX_METHOD_IMPL_VAL  0xffff
 #define METHOD_IMPL_ATTRIBUTE_AGGRESSIVE_INLINING  0x0100
+#define METHOD_IMPL_ATTRIBUTE_REABSTRACTION        0x0200
```

Comment:
These flags come from ECMA 335 - we can't just add new flags here.

You can add an `is_reabstracted` bitfield here:
https://github.com/mono/mono/blob/master/mono/metadata/class-internals.h#L80